CHANGELOG.md

```diff
@@ -14,9 +14,9 @@
 - Aggregate pod status to reduce notification noise [\#212](https://github.com/infracloudio/botkube/issues/212)
 - Add support to monitor custom resources [\#200](https://github.com/infracloudio/botkube/issues/200)
 - Limit kubectl commands [\#183](https://github.com/infracloudio/botkube/issues/183)
-- Swich to github actions for CI builds [\#179](https://github.com/infracloudio/botkube/issues/179)
+- Switch to github actions for CI builds [\#179](https://github.com/infracloudio/botkube/issues/179)
 - Setting default namespace while executing kubectl commands [\#176](https://github.com/infracloudio/botkube/issues/176)
-- Add Microsoft Teams suppport [\#60](https://github.com/infracloudio/botkube/issues/60)
+- Add Microsoft Teams support [\#60](https://github.com/infracloudio/botkube/issues/60)
```

Comment:
@sanketsudake this will be altered next time we run changelog_generator, the correct fix would be to fix the issues/PRs names so that next time when it tried to generate changelog, it will get correct names

Reply:
I have corrected typos in respective issues and PRs referred from changelog. Also, keeping this change so that Github action passes.
@PrasadG193 Please let me know if this helps.

Reply:
Sounds good. Thanks @sanketsudake